src/main/webapp/js/maps-page-loader.js

```diff
@@ -0,0 +1,259 @@
+						{
+							"color": "#283d6a"
+						}
+					]
+				},
+				{
+					"featureType": "poi",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#6f9ba5"
+						}
+					]
+				},
+				{
+					"featureType": "poi",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "poi.park",
+					"elementType": "geometry.fill",
+					"stylers": [
+						{
+							"color": "#023e58"
+						}
+					]
+				},
+				{
+					"featureType": "poi.park",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#3C7680"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#304a7d"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#98a5be"
+						}
+					]
+				},
+				{
+					"featureType": "road",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#2c6675"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "geometry.stroke",
+					"stylers": [
+						{
+							"color": "#255763"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#b0d5ce"
+						}
+					]
+				},
+				{
+					"featureType": "road.highway",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#023e58"
+						}
+					]
+				},
+				{
+					"featureType": "transit",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#98a5be"
+						}
+					]
+				},
+				{
+					"featureType": "transit",
+					"elementType": "labels.text.stroke",
+					"stylers": [
+						{
+							"color": "#1d2c4d"
+						}
+					]
+				},
+				{
+					"featureType": "transit.line",
+					"elementType": "geometry.fill",
+					"stylers": [
+						{
+							"color": "#283d6a"
+						}
+					]
+				},
+				{
+					"featureType": "transit.station",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#3a4762"
+						}
+					]
+				},
+				{
+					"featureType": "water",
+					"elementType": "geometry",
+					"stylers": [
+						{
+							"color": "#0e1626"
+						}
+					]
+				},
+				{
+					"featureType": "water",
+					"elementType": "labels.text.fill",
+					"stylers": [
+						{
+							"color": "#4e6d70"
+						}
+					]
+				}
+			],
+			{name: 'Night'});
+
+	// Create a map object, and include the MapTypeId to add
+	// to the map type control.
+	var map = new google.maps.Map(document.getElementById('map'), {
+		center: {lat: 1.2764558, lng: 103.7996469},
```

Comment:
The separation of dynamic logical code and static configuration comment applies here as well. And similarly, just fyi.

Reply:
Do I separate the center into a separate .json file? But that would mean that the .json file would only contain this value and nothing else...would that be recommended? Or maybe I can declare it as a constant (like what you recommended for element ID 'map' above? May I ask which is better?

Reply:
I think that leaving this here is fine (: Its an unnecessary over-engineering, but my comment was just meant to be educational haha like as a project gets bigger, it would be good to start separating config and code. But this is a small thing, so it is really not necessary.